source/change-streams/tests/unified/change-streams.yml

```diff
@@ -226,3 +226,27 @@ tests:
                 comment: { $$exists: false }
               commandName: getMore
               databaseName: *database0
+  
+  - description: "to field is set in a rename change event"
+    runOnRequirements:
+      - minServerVersion: "4.0.1"
+    operations:
+      - name: createChangeStream
+        object: *collection0
+        arguments:
+          pipeline: []
+        saveResultAsEntity: &changeStream0 changeStream0
+      - name: rename
```

Comment:
Oh, I didn't realize there was already another PR that added a `rename` operation. And this test file uses `schemaVersion: "1.0"`.

I think that can change our approach in https://github.com/mongodb/specifications/pull/1156#discussion_r839959883 and we can safely just start adding `rename` to test files without bumping the schema version, as any driver implementing a downstream ticket for [DRIVERS-1343](https://jira.mongodb.org/browse/DRIVERS-1343) is going to encounter this already.